src/FirstSteps.Tests/RPG/Adventures/MineTests.cs

```diff
@@ -38,5 +38,30 @@ public void Dwarf_Should_Gain_Three_Coins_When_Numbers_Are_Odd_Otherwise_Should_
 
             dwarf.Coins.Should().Be(expectedCoins);
         }
+        [Theory]
+        [InlineData(1, 1)]
+        [InlineData(4, 2)]
+        [InlineData(5, 3)]
+        [InlineData(8, 4)]
+        [InlineData(10, 5)]
+        [InlineData(13, 6)]
+        [InlineData(14, 7)]
+        [InlineData(17, 8)]
+        [InlineData(18, 9)]
+        [InlineData(21, 10)]
+        [InlineData(22, 11)]
+        [InlineData(25, 12)]
+        [InlineData(25, 13)]
+        public void Dwarf_Should_Gain_Three_Coins_When_Numbers_Are_Odd_Or_Plus_2_Coins_When_Divide_By_Five_When_Number_Is_13_Gain_0_Coins_Otherwise_Gain_1_Coin(int expectedCoins, int visitCounter)
```

Comment:
Try to run unit tests locally in Visual Studio, you will see that two tests fail and maybe it will be helpful.